app/controllers/hyrax/pages_controller.rb

```diff
@@ -0,0 +1,54 @@
+module Hyrax
+  # Shows the about and help page
+  class PagesController < ApplicationController
```

Comment:
Why do we need a local copy of this? Are we introducing customization that requires it?